src/main/java/com/deavensoft/paketomat/center/model/Paketomat.java

```diff
@@ -1,17 +1,30 @@
 package com.deavensoft.paketomat.center.model;
 
+import com.fasterxml.jackson.annotation.JsonProperty;
 import lombok.*;
 
+import javax.persistence.*;
 import java.util.ArrayList;
-import java.util.UUID;
 
-@Data
+@Getter
+@Setter
 @AllArgsConstructor
 @NoArgsConstructor
+@Entity
+@Table(name = "paketomats", schema = "public")
 public class Paketomat {
-
-    private UUID id;
-    private City city;
-    private int size;
-    private ArrayList<Package> packages;
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @Column(name = "id")
+    private Long id;
+    @Column(name = "city")
+    private Long city;
+    @Transient
+    private  static final int SIZE = 5;
+    @Transient
```

Comment:
We have @Transient here because we think it is better to save lists of packages that will arrive to Paketomat like this, and not add it to the Database, since this is something that will be changed a lot.